### DISCUSSION TITLE
Can we Have a Read Tab ?

### DISCUSSION THREAD

**Discussion by Yummy-Yums** [dailydotdev/daily#1154] [Feature Requests]
Hello, 

Please can we have a read Tab for list of posts read ?

**Comment by jeffchavez-dev** [comment]
Great idea.

**Reply by Yummy-Yums** [reply]
i just read a post and wanted to share but couldn't find it after i refreshed the page

**Comment by nimrodkra** [comment]
There’s a page for it called “History”. On desktop you can find it on the
left sidebar and on mobile you can find it on the header (need to scroll
horizontally to the right to see it). Either way you can find it via this
link: https://app.daily.dev/history

**Reply by Yummy-Yums** [reply]
ohk, i can see it now, thanks
